pkg/controller/authentication/containers.go

```diff
@@ -654,7 +654,7 @@ func buildIdentityProviderContainer(instance *operatorv1alpha1.Authentication, i
 		},
 	}
 
-	idpEnvVarList := []string{"NODE_ENV", "LOG_LEVEL_IDPROVIDER", "LOG_LEVEL_MW", "PROVIDER_ISSUER_URL", "IDTOKEN_LIFETIME", "ROKS_ENABLED", "ROKS_URL", "OS_TOKEN_LENGTH", "LIBERTY_TOKEN_LENGTH",
+	idpEnvVarList := []string{"NODE_ENV", "LOG_LEVEL_IDPROVIDER", "LOG_LEVEL_MW", "PROVIDER_ISSUER_URL", "PREFERRED_LOGIN", "IDTOKEN_LIFETIME", "ROKS_ENABLED", "ROKS_URL", "OS_TOKEN_LENGTH", "LIBERTY_TOKEN_LENGTH",
 		"IDENTITY_PROVIDER_URL", "BASE_AUTH_URL", "BASE_OIDC_URL", "SCOPE_CLAIM", "OIDC_ISSUER_URL", "HTTP_ONLY"}
```

Comment:
@rashmi43 Don't you need XFRAME_DOMAIN here?

Reply:
No I do not